index.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
 		'function-linear-gradient-no-nonstandard-direction': true,
 		'function-no-unknown': true,
 		'keyframe-declaration-no-important': true,
+		'keyframe-block-no-duplicate-selectors': true,
```

Comment:
[ask] Should be in alphabetical order? (it's trivial though)

```suggestion
		'keyframe-block-no-duplicate-selectors': true,
		'keyframe-declaration-no-important': true,
```

Reply:
I really need to practice my alphabet 😆